<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiView</class>
 <widget class="QMainWindow" name="UiView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IanniX â€” View</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 11px &quot;Arial&quot;;
}

QWidget#centralwidget  {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(74, 74, 74, 255), stop:1 rgba(22, 22, 22, 255));
}

QDockWidget {
	color: white;
	font: 10px;
}
QDockWidget::title {
     text-align: center;
     background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(74, 74, 74, 255), stop:1 rgba(22, 22, 22, 255));
     padding-left: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="UiRender" name="render" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_all"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionDrawFreeCurve"/>
    <addaction name="actionDrawPointCurve"/>
    <addaction name="actionDrawTriggers"/>
    <addaction name="actionAddCircleCurve"/>
    <addaction name="actionAddFreeCursor"/>
    <addaction name="separator"/>
    <addaction name="actionSync"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_all"/>
    <addaction name="separator"/>
    <addaction name="actionImoprt_SVG"/>
    <addaction name="actionImoprt_Image"/>
    <addaction name="actionImoprt_Text"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="actionRemove"/>
    <addaction name="actionDuplicateScore"/>
    <addaction name="separator"/>
    <addaction name="actionReloadScript"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuGrid_size">
     <property name="title">
      <string>Grid setup</string>
     </property>
     <addaction name="action10Seconds"/>
     <addaction name="action1second"/>
     <addaction name="action500Milliseconds"/>
     <addaction name="action100Milliseconds"/>
     <addaction name="action10Milliseconds"/>
     <addaction name="actionCustomValue"/>
     <addaction name="separator"/>
     <addaction name="actionOpaque"/>
     <addaction name="action75_opaque"/>
     <addaction name="action50_opaque"/>
     <addaction name="action25_opaque"/>
     <addaction name="action10_opaque"/>
    </widget>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Inspector"/>
    <addaction name="actionToggle_Transport"/>
    <addaction name="separator"/>
    <addaction name="actionGrid"/>
    <addaction name="actionSnapGrid"/>
    <addaction name="menuGrid_size"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_initial"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionToggle_Autosize"/>
    <addaction name="separator"/>
    <addaction name="actionShowEditor"/>
    <addaction name="actionLight"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="actionPlay_pause"/>
    <addaction name="actionFast_rewind"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuPlayback"/>
  </widget>
  <widget class="QDockWidget" name="transportDock">
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>65</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Transport</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="UiTransport" name="transport"/>
  </widget>
  <widget class="QDockWidget" name="inspecteurDock">
   <property name="minimumSize">
    <size>
     <width>355</width>
     <height>42</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="UiInspector" name="inspector"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionDrawTriggers"/>
   <addaction name="actionDrawFreeCurve"/>
   <addaction name="actionDrawPointCurve"/>
   <addaction name="actionAddCircleCurve"/>
   <addaction name="actionAddFreeCursor"/>
   <addaction name="separator"/>
   <addaction name="actionLockPos"/>
   <addaction name="actionToggleLabel"/>
   <addaction name="separator"/>
   <addaction name="actionGrid"/>
   <addaction name="actionSnapGrid"/>
   <addaction name="separator"/>
   <addaction name="actionToggle_Inspector"/>
   <addaction name="actionToggle_Transport"/>
   <addaction name="actionFullscreen"/>
   <addaction name="separator"/>
   <addaction name="actionLight"/>
   <addaction name="separator"/>
   <addaction name="actionPatchesFolder"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New score</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open project folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save current score</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_fullscreen.png</normaloff>:/actions/res_icon_fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToggle_Inspector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_inspector.png</normaloff>:/actions/res_icon_inspector.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Inspector</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionToggle_Transport">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_transport.png</normaloff>:/actions/res_icon_transport.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Transport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionPlay_pause">
   <property name="text">
    <string>Play/pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionFast_rewind">
   <property name="text">
    <string>Fast rewind</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_grid.png</normaloff>:/actions/res_icon_grid.png</iconset>
   </property>
   <property name="text">
    <string>Grid &amp;&amp; axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionToggle_Autosize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autoresize object</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_initial">
   <property name="text">
    <string>Initial zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ã€</string>
   </property>
  </action>
  <action name="actionDrawFreeCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_curveFree.png</normaloff>:/actions/res_icon_curveFree.png</iconset>
   </property>
   <property name="text">
    <string>Draw freehand curve</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionDrawPointCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_curvePoint.png</normaloff>:/actions/res_icon_curvePoint.png</iconset>
   </property>
   <property name="text">
    <string>Draw point-by-point curve</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionDrawTriggers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_trigger.png</normaloff>:/actions/res_icon_trigger.png</iconset>
   </property>
   <property name="text">
    <string>Draw triggers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionAddFreeCursor">
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_cursor.png</normaloff>:/actions/res_icon_cursor.png</iconset>
   </property>
   <property name="text">
    <string>Add a cursor</string>
   </property>
  </action>
  <action name="action10Seconds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 seconds</string>
   </property>
  </action>
  <action name="action1second">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 second</string>
   </property>
  </action>
  <action name="action500Milliseconds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>500 milliseconds</string>
   </property>
  </action>
  <action name="action100Milliseconds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100 milliseconds</string>
   </property>
  </action>
  <action name="action10Milliseconds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 milliseconds</string>
   </property>
  </action>
  <action name="actionCustomValue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom value</string>
   </property>
  </action>
  <action name="actionOpaque">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Opaque</string>
   </property>
  </action>
  <action name="action75_opaque">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>75% opaque</string>
   </property>
  </action>
  <action name="action50_opaque">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50% opaque</string>
   </property>
  </action>
  <action name="action25_opaque">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>25% opaque</string>
   </property>
  </action>
  <action name="action10_opaque">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10% opaque</string>
   </property>
  </action>
  <action name="actionAddCircleCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_curveCircle.png</normaloff>:/actions/res_icon_curveCircle.png</iconset>
   </property>
   <property name="text">
    <string>Draw circle curves</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_help.png</normaloff>:/actions/res_icon_help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSnapGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_grid_snap.png</normaloff>:/actions/res_icon_grid_snap.png</iconset>
   </property>
   <property name="text">
    <string>Snap to grid</string>
   </property>
   <property name="shortcut">
    <string>Shift+G</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+S</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename score</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+R</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove score</string>
   </property>
  </action>
  <action name="actionDuplicateScore">
   <property name="text">
    <string>Duplicate score</string>
   </property>
  </action>
  <action name="actionLockPos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_lock.png</normaloff>:/actions/res_icon_lock.png</iconset>
   </property>
   <property name="text">
    <string>Lock objects position</string>
   </property>
  </action>
  <action name="actionToggleLabel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_label.png</normaloff>:/actions/res_icon_label.png</iconset>
   </property>
   <property name="text">
    <string>Toggle objects labels</string>
   </property>
  </action>
  <action name="actionPatchesFolder">
   <property name="text">
    <string>Open patches folder</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About IanniX</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>Send a snapshot of score to sync output</string>
   </property>
  </action>
  <action name="actionImoprt_SVG">
   <property name="text">
    <string>Import SVGâ€¦</string>
   </property>
  </action>
  <action name="actionImoprt_Image">
   <property name="text">
    <string>Import imageâ€¦</string>
   </property>
  </action>
  <action name="actionImoprt_Text">
   <property name="text">
    <string>Import glyphâ€¦</string>
   </property>
  </action>
  <action name="actionShowEditor">
   <property name="text">
    <string>Show script editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionReloadScript">
   <property name="text">
    <string>Reload last script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IanniX.qrc">
     <normaloff>:/actions/res_icon_light.png</normaloff>:/actions/res_icon_light.png</iconset>
   </property>
   <property name="text">
    <string>Lighter color theme</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UiRender</class>
   <extends>QWidget</extends>
   <header>uirender.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UiTransport</class>
   <extends>QWidget</extends>
   <header>uitransport.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UiInspector</class>
   <extends>QWidget</extends>
   <header>uiinspector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="IanniX.qrc"/>
 </resources>
 <connections/>
</ui>
